storage/cache/subtree_cache.go

```diff
@@ -19,8 +19,21 @@ type GetSubtreeFunc func(id storage.NodeID) (*storage.SubtreeProto, error)
 // SetSubtreeFunc describes a function which can store a Subtree into storage.
 type SetSubtreesFunc func(s []*storage.SubtreeProto) error
 
+// strataInfo represents a single statum across the tree.
+// It it used inside the SubtreeCache to determine which Subtree prefix should
+// be used for a given NodeID.
+type strataInfo struct {
+	// prefixBytes is the number of prefix bytes above this stratum.
+	prefixBytes int
+	// depth is the depth of this stratum.
```

Comment:
Neither, it's the number of levels in the stratum - I've updated the comment since it wasn't clear.